src/main/java/com/ecommerce/service/AccountServiceDefault.java

```diff
@@ -23,8 +24,14 @@ public Account getAccountById(UUID id) {
   }
 
   @Override
-  public void createAccount(final Account account) {
-    repository.save(getAccount(account));
+  public void  createAccount(final Account account) throws ConnectionException {
+	  try {
+     repository.save(getAccount(account));
+  }
+	  catch (Exception e) {
+		  throw new ConnectionException("Can´t create account at this time.  Please try again later");	
```

Comment:
Dentro del mensaje de la excepcion veo que hay mas de un espacio en `this time.  Please try` por favor corregirlo para que quede algo como `this time. Please try`